css/style.css

```diff
@@ -0,0 +1,6 @@
+#image {
+    max-width: 100%;
+    display: block;
+    height: auto;
+    margin: 0 auto;
```

Comment:
@studiosnabeel 😊Please use CSS best practice by leaving only 2 spaces🤗. This way your code will look more professional and will be up to standard 😉.